pkg/provider/browser/browser_test.go

```diff
@@ -39,7 +39,8 @@ func TestValidate(t *testing.T) {
 	client, err := New(account)
 	assert.Nil(t, err)
 	loginDetails := &creds.LoginDetails{
-		URL: "https://google.com/",
+		URL:             "https://google.com/",
+		DownloadBrowser: true,
```

Comment:
is there a way to test fail without download?

Reply:
In a word, yes. It should be easy to copy/paste this test without the flag set and check if the error is not nil since I believe that's how the error is being propagated up to the user.

Reply:
Okay, so I've added the test, though the change was largely than I had though it'll be. 

I had added a new config parameter to technically allow one to set the directory that Playwright installs into though I've purposely not documented.

I did this way because otherwise my test was going to be sensitive if the previous test ran and installed playwright before this test which introduces all sorts of headaches.

So instead, you technically can now choose where you download the drivers to, defaulting to the default location if you don't care.

Reply:
I left the existing test as it is to download the drivers to the default location  though an argument could be made it shouldn't.